spec/features/any_user_can_see_beer_list_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+feature "any user is able to see" do 
+
+  before(:each) do 
+  category = Category.create({name: "Lager",
+                                 description: "Yum"
+                                 })
+
+  beer1 = {name: "Titan",
+          state: true,
+          description: "aggressively hopped ipa",
+          price: 350}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.